CIP-onchain-cert-metadata/README.md

```diff
@@ -0,0 +1,489 @@
+---
+CIP: ?
+Title: On-chain Certification Metadata Standard
+Category: Metadata
+Status: Proposed
+Authors:
+    - Romain Soulat <romain.soulat@iohk.io>
+    - Nicolas Jeannerod <nicolas.jeannerod@tweag.io>
+    - Mathieu Montin <mathieu.montin@tweag.io>
+    - M. Ali Modiri <mixaxim@gimbalabs.io>
+    - Simon Thompson <simon.thompson@iohk.io>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/499
+    - https://github.com/input-output-hk/Certification-working-group/pull/18
+    - https://docs.google.com/document/d/1vC4cS-gMLIvgduSh5Fipn_lU_0Ou0NY4X_a9lWJz2B8
+Created: 2023-02-23
+License: CC-BY-4.0
+---
+
+
+## Abstract
+
+Certification is the process of providing various kinds of assurance of DApps through the provision of verifiable, trustworthy metadata linking to relevant evidence of assurance, such as audit reports, test run data, formal verification proofs etc.
+
+Currently, certification does not have a standardised way to be stored and presented to all stakeholders (DApp developer, DApp stores, end-users, auditors, ...) in an immutable, persistent and verifiable way.
+
+This CIP descibes a standardised method for certificates to be published and stored on-chain and for stakeholders to be able to verify the different claims of this certificate.
+
+This proposal describes how certification metadata can be designed for DApp registration. It first describes the requirements for DApp certification, and then sets out the options for its inclusion in CIP-72.
```

Comment:
This is an ancient artifact for when we were considering adding this CIP directly into the CIP 72 for registration. We've decided it would be better to be a separate one. I'll change the sentence.

Reply:
Was looking for change but don't see it... @RSoulatIOHK I guess you are still planning on updating this?

Reply:
OK, resolving since it looks like offending sentence has been corrected to
> This CIP describes how a certification certificate can be registered on-chain and how to anchor it to a more detailed off-chain certification report.